utility/src/main/java/org/oppia/util/data/DataProviders.kt

```diff
@@ -20,7 +21,8 @@ import org.oppia.util.threading.BackgroundDispatcher
 @Singleton
 class DataProviders @Inject constructor(
   @BackgroundDispatcher private val backgroundDispatcher: CoroutineDispatcher,
-  private val asyncDataSubscriptionManager: AsyncDataSubscriptionManager
+  private val asyncDataSubscriptionManager: AsyncDataSubscriptionManager,
+  private val crashlyticsWrapper: CrashlyticsWrapper
```

Comment:
A note on naming: while it a wrapper, if you change this to be an interface (per a later comment of mine) it may be clearer to name it based on its anticipated functionality, rather than its implementation. It seems to be a CrashLogger, and right now we're using Crashlytics as the implementation for that logger. I suggest naming it based on its usage rather than its implementation.

Reply:
Renamed to crashLogger

Reply:
Sorry, I left a comment recently suggesting we use ExceptionLogger instead. I should have thought about this a bit more. It occurred to me now that this isn't actually meant for logging crashes, but instead cases when the app doesn't crash. Even if we add fatal exception logging later, 'ExceptionLogger' still seems a bit more generic.